File: Android/app/src/main/java/com/example/childcare/models/Service.java
package com.example.childcare.models;

public class Service {
    private int serviceID;
    private String name;
    private String description;
    private double price;
    private int durationMinutes;
    private String imageUrl;

    public int getServiceID() { return serviceID; }
    public String getName() { return name; }
    public String getDescription() { return description; }
    public double getPrice() { return price; }
    public int getDurationMinutes() { return durationMinutes; }
    public String getImageUrl() { return imageUrl; }
}

File: API/ChildCare.Api/Models/Service.cs
using System;
using System.Collections.Generic;

namespace ChildCare.Api.Models;

public partial class Service
{
    public int ServiceId { get; set; }

    public string Name { get; set; } = null!;

    public string? Description { get; set; }

    public decimal Price { get; set; }

    public int DurationMinutes { get; set; }

    public DateTime? CreatedAt { get; set; }

    public string? ImageUrl { get; set; }

    public virtual ICollection<Appointment> Appointments { get; set; } = new List<Appointment>();
}
